nextmv/cloud/application.py

```diff
@@ -646,6 +653,28 @@ def run_input(
 
         return download_response.json()
 
+    def run_logs(
+        self,
+        run_id: str,
+    ) -> Dict[str, Any]:
+        """
+        Get the logs of a run.
+
+        Args:
+            run_id: ID of the run.
+
+        Returns:
+            Logs of the run.
+
+        Raises:
+            requests.HTTPError: If the response status code is not 2xx.
+        """
+        response = self.client.request(
+            method="GET",
+            endpoint=f"{self.endpoint}/runs/{run_id}/logs",
+        )
+        return RunLog.from_dict(response.json())
```

Comment:
I think the pattern we are following so far is to return the actual class. People can then do whatever they want with it, like converting it to a dict. Up to you.

Reply:
Good catch! Fixed it. :hugs: